### PR TITLE
[DOC] Action classification explanation part

### PR DESCRIPTION
**Pull request by JihwanEom**
## Preview
image

### CODE REVIEW

**Review by harimkang — APPROVED** [review]
When are tables updated?

**Review by supersoob — APPROVED** [review]

**Comment by JihwanEom** [comment]
> When are tables updated?

Will be updated after merging #1738

File: docs/source/explanation/algorithms/action/action_classification.rst [hunk 1]
```diff
@@ -1,14 +1,47 @@
-Action Classificaiton
-=====================
+Action Recognition
```
Comment (jaegukhyun) [inline docs/source/explanation/algorithms/action/action_classification.rst:1]: We normally use action recognition term to call both action classification and action detection. Therefore this should be action classification

**Review by jaegukhyun — CHANGES_REQUESTED** [review]
I think action classification is right not action recognition

**Review by jaegukhyun — APPROVED** [review]

**Comment by goodsong81** [comment]
@JihwanEom Could you refine this in another PR? Emily requested quick merge to address advices from Geti UX/doc team.

File: docs/source/explanation/algorithms/action/action_classification.rst (hunk 1 above)
Comment (kprokofi) [reply docs/source/explanation/algorithms/action/action_classification.rst:1]: agree

File: docs/source/explanation/algorithms/action/action_classification.rst [hunk 2]
```diff
@@ -1,14 +1,47 @@
...
+==================
+
+Action recognition is a problem of identifying the action that is being performed in a video. The input to the algorithm is a sequence of video frames, and the output is a label indicating the action that is being performed.
+
+For supervised learning we use the following algorithms components:
+
+- ``Augmentations``: We use standard data augmentations for videos, including random resizing and random cropping, horizontal flipping. We randomly sample a segment of frames from each video during training.
+
+- ``Optimizer``: We use the Adam with weight decay fix (AdamW) optimizer.
```
Comment (kprokofi) [inline docs/source/explanation/algorithms/action/action_classification.rst:10]: please, add a link. Some users can be not familiar with AdamW or even Adam and it is just a good manner to have a link for components

File: docs/source/explanation/algorithms/action/action_classification.rst [hunk 3]
```diff
@@ -1,14 +1,47 @@
...
+
+- ``Learning rate schedule``: We use a step learning rate schedule, where the learning rate is reduced by a factor of 10 after a fixed number of epochs. We also use the Linear Warmup technique to gradually increase the learning rate at the beginning of training.
```
Comment (kprokofi) [inline docs/source/explanation/algorithms/action/action_classification.rst:12]: I would suggest to highlight all the components with bold text. For example, **step learning rate schedule**, **Linear Warmup**, etc.

File: docs/source/explanation/algorithms/action/action_classification.rst [hunk 4]
```diff
@@ -1,14 +1,47 @@
...
+
+- ``Loss function``: We use the Cross-Entropy Loss as the loss function.
```
Comment (kprokofi) [inline docs/source/explanation/algorithms/action/action_classification.rst:14]: Please, provide a link (we have it for other tasks, for example, classification

File: docs/source/explanation/algorithms/action/action_classification.rst [hunk 5]
```diff
@@ -1,14 +1,47 @@
...
 
 **************
 Dataset Format
 **************
 
+We support the popular action classification formats, such as `Jester <https://developer.qualcomm.com/software/ai-datasets/jester>`_, `HMDB51 <https://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/>`_, `UCF101 <https://www.crcv.ucf.edu/data/UCF101.php>`_. Specifically, these formats will be converted into our `internal representation <https://github.com/openvinotoolkit/training_extensions/tree/develop/tests/assets/cvat_dataset/action_classification/train>`_ using the `Datumaro <https://github.com/openvinotoolkit/datumaro>`_ dataset handler.
+
+The names of the annotations files and the overall dataset structure should be the same as the original dataset.
+
+Refer to our tutorial for more information on how to train, validate, and optimize action classification models.
```
Comment (kprokofi) [inline docs/source/explanation/algorithms/action/action_classification.rst:24]: Please, provide a link to tutorials

File: docs/source/explanation/algorithms/action/action_classification.rst [hunk 6]
```diff
@@ -1,14 +1,47 @@
...
+
 ******
 Models
 ******
 
-********************
-Incremental Learning
-********************
\ No newline at end of file
+We support `X3D <https://arxiv.org/abs/2004.04730>`_ for action classification. X3D is a deep learning model that was proposed in the paper "X3D: Expanding Architectures for Efficient Video Recognition" by Christoph Feichtenhofer. The model is an extension of the popular 2D convolutional neural network (CNN) architectures to the 3D domain, allowing it to efficiently process spatiotemporal information in videos.
```
Comment (kprokofi) [inline docs/source/explanation/algorithms/action/action_classification.rst:30]: "X3D is a deep learning model that was proposed in the paper "X3D: Expanding Architectures for Efficient Video Recognition" by Christoph Feichtenhofer" - I think it is redundant information since you already provided a link to the article

**Review by kprokofi — COMMENTED** [review]
Thank you very much for your input in docs! I left some comments, please, reflect them in separate PR
